src/ClientConfiguration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Predis;
+
+class ClientConfiguration
+{
+    /**
+     * @var array{modules: array}|string[][]
+     */
+    private static $config = [
+        'modules' => [
+            ['name' => 'Json', 'commandPrefix' => 'JSON'],
+            ['name' => 'Graph', 'commandPrefix' => 'GRAPH'],
```

Comment:
Should these be here if this is json only? I realize we need them for the future.. but IMHO for now, they don't necessarily belong in this PR.

Reply:
Yeah, I agree. Not necessary at least now